garfield/garfield/settings.py

```diff
@@ -21,6 +21,8 @@
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = 'CHANGE ME'
+SID= 'TWILIO SID'
```

Comment:
We keep these credentials in `garfield/garfield/local.py` so they can be customized by instance. We try to keep `settings.py` global for all instances and `local.py` for configuration settings like this.

Checkout `local.sample` for the namespace.